pkg/collector/cluster.go

```diff
@@ -121,7 +121,7 @@ func (c *ClusterCollector) Get() ([]map[string]interface{}, error) {
 		log.Debug().Msgf("Retrieving: %s.%s.%s", g.Resource, g.Version, g.Group)
 		rs, err := ri.List(context.Background(), metav1.ListOptions{})
 		if err != nil {
-			log.Debug().Msgf("Failed to retrieve: %s: %s", g, err)
+			log.Warn().Msgf("Failed to retrieve: %s: %s", g, err)
```

Comment:
I don't think this is desirable, and that we actually previously lowered the log-level of this. It will be a common occurrence that when run on K8S versions that don't already support the resource version, the cluster collector won't be able to retrieve those. And we don't want to warn about that by default.

Reply:
ok removed